zcash_client_sqlite/src/wallet/init/migrations/transparent_gap_limit_handling.rs

```diff
@@ -75,6 +80,7 @@ impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
             ALTER TABLE addresses ADD COLUMN transparent_child_index INTEGER;
             ALTER TABLE addresses ADD COLUMN exposed_at_height INTEGER;
             ALTER TABLE addresses ADD COLUMN receiver_flags INTEGER;
+            ALTER TABLE addresses ADD COLUMN transparent_receiver_next_check_time INTEGER;
```

Comment:
We never fill anything into this column, so why add it to the original table? Is it present just to make existing updated queries work during this migration? If so, document this. If not, remove it.